docs/source/installation.rst

```diff
@@ -136,52 +136,50 @@ Step 3: Install ``torch`` and ``torchvision`` with CPU or GPU Support
 
     .. tabs::
 
-        .. tab:: CPU Support
-
-            If you plan to run ``ivadomed`` on CPU only, install PyTorch per instructions provided below for your specific operating system:
+        .. group-tab:: PyPI Installation
 
             .. tabs::
 
-                .. tab:: Windows/Linux
+                .. tab:: Nvidia GPU Support
 
-                    .. code::
+                    ``ivadomed`` requires CUDA11 to execute properly. If you have a nvidia GPU, try to look up its Cuda Compute Score `here <https://developer.nvidia.com/cuda-gpus>`__, which needs to be > 3.5 to support CUDA11. Then, make sure to upgrade to nvidia driver to be at least v450+ or newer.
 
-                       pip install torch==1.8.0+cpu torchvision==0.9.0+cpu --find-links https://download.pytorch.org/whl/torch_stable.html
+                    You can use ``nvidia-smi`` in both Linux and Windows to check for driver CUDA Version listed at the top right of the output console. On windows, follow `this example guide here until Step 4 <https://wiki.tuflow.com/index.php?title=DOS_GPU_Usage>`__. On Linux, simply type in ``nvidia-smi`` in any console to see the output.
```

Comment:
I think this Windows tutorial link might be out of date? According to [this StackOverflow comment](https://stackoverflow.com/a/57100016), the location of `nvidia-smi` seems to vary depending on how recent the version is. 

So, we might want to word it like this:

```suggestion
                    You can use ``nvidia-smi`` in both Linux and Windows to check for driver CUDA Version listed at the top right of the output console. On Linux, simply type in ``nvidia-smi`` in any console to see the output. On windows, you will need to locate the `nvidia-smi.exe` tool by following the instructions on `this page <https://stackoverflow.com/a/57100016>`__. 
```